src/changefile/promptForChange.ts

```diff
@@ -36,7 +36,7 @@ export async function promptForChange(options: BeachballOptions) {
       message: 'Change type',
       choices: [
         ...(showPrereleaseOption ? [{ value: 'prerelease', title: ' [1mPrerelease[22m - bump prerelease version' }] : []),
-        { value: 'patch', title: ' [1mPatch[22m      - bug fixes; no backwards incompatible changes.' },
+        { value: 'patch', title: ' [1mPatch[22m      - bug fixes; backwards compatible changes.' },
         { value: 'minor', title: ' [1mMinor[22m      - small feature; backwards compatible changes.' },
```

Comment:
Thanks for the contribution but now that you point this out, I actually think both the patch and minor descriptions are unclear. I'd suggest something like this instead:
```suggestion
        { value: 'patch', title: ' [1mPatch[22m      - bug fixes; no API changes.' },
        { value: 'minor', title: ' [1mMinor[22m      - small feature; backwards compatible API changes.' },
```
@xugao and/or @kenotron what do you think?

Reply:
Agreed!